.github/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+## Scope and purpose
+
+Fixes #<!-- ISSUE-ID -->. Dependent on #<!-- PULL-REQUEST-ID -->. <!-- Reference: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue -->
+
+Add a description of what this PR does and why it is needed. If a linked ticket(s) fully
+cover it, you can omit this.
+
+<!-- remove things that do not apply -->
+### This pull request
+* adds/changes/removes a dependency
+* changes the database
+* changes the API
+
+
+## Contributor Checklist
+
+Every pull request should have this checklist filled out, no matter how small it is.
+More information about contributing to NAV can be found in the
+[Hacker's guide to NAV](https://nav.readthedocs.io/en/latest/hacking/hacking.html#hacker-s-guide-to-nav).
+
+<!-- Add an "X" inside the brackets to confirm -->
+<!-- If not checking one or more of the boxes, please explain why below each. -->
+
+* [ ] Added a changelog fragment for [towncrier](https://nav.readthedocs.io/en/latest/hacking/hacking.html#adding-a-changelog-entry)
```

Comment:
Should we explain somewhere that if the contributor deems that this change is purely internal and doesn't deserve a news item to be published to end users, that the `nonews` label can be added (are outside contributors allowed to add labels?)?

Reply:
"Anyone with triage access to a repository can apply and dismiss labels." (See: https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization#permissions-for-each-role)

So most outside contributors won't be able to do so, so I feel like we can omit that